Formula/minimesos.rb

```diff
@@ -0,0 +1,16 @@
+class Minimesos < Formula
+  desc "Testing infrastructure for Mesos frameworks"
+  homepage "https://minimesos.org/"
+  url "https://github.com/ContainerSolutions/minimesos/archive/0.10.2.tar.gz"
+  sha256 "16b198078c9384426f8849b08e92e173816c4d6626331bc6569aa5de89dffbdd"
+
+  depends_on "docker"
+
+  def install
+    bin.install "bin/minimesos"
+  end
+
+  test do
+    system "test -e #{bin}/minimesos && true || false"
```

Comment:
Please run the executable and `assert_match` on the version or something, thanks.